engineering/conventions/go.md

```diff
@@ -7,17 +7,17 @@ This guide documents development conventions for Go at source{d}. Check [general
 
 ## Supported Go Versions
 
-* Our libraries support latest two stable major versions of Go (e.g. 1.10.x and 1.11.x). Both versions should be included in Travis CI.
+* Our **libraries** support latest two stable major versions of Go (1.11.x and 1.12.x). Both versions should be included in Travis CI.
   * Some of our libraries may support more versions, specially when there is wide-adoption outside source{d}, (e.g. [go-git](https://github.com/src-d/go-git) which currently supports three.
-* Our applications support only latest [stable](https://golang.org/dl/#stable) Go version.
+* Our **applications** support only latest [stable](https://golang.org/dl/#stable) Go version (1.12.x).
 
 ## Dependency Management
 
-* If your project is an application:
-  * Use [dep](https://github.com/golang/dep) to manage dependencies.
-  * Check in the `vendor` directory into git.
-* If your project is a library:
+* If your project is a **library**:
   * Use [gopkg.in](http://labix.org/gopkg.in) to provide versioned imports.
```

Comment:
Use of `gopkg.in` is not mutually exclusive with Go modules. I think we should support _both_ for libraries. With modules, `gopkg.in` is mostly redundant, but it allows existing import paths to keep working. But within the library itself, we should manage dependencies with modules IMO.

Reply:
This is discussion is planned for a separate PR :) I will open a new one right after this one to avoid conflicts.